pkg/dataconn/client.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/sirupsen/logrus"
 
+	"github.com/longhorn/longhorn-engine/pkg/types"
 	journal "github.com/longhorn/sparse-tools/stats"
```

Comment:
```suggestion
	journal "github.com/longhorn/sparse-tools/stats"

    "github.com/longhorn/longhorn-engine/pkg/types"
```

Reply:
Looks like it automerged, but I'll create a quick PR to fix this.